es/django_forms/README.md

```diff
@@ -126,10 +126,10 @@ Para crear un nuevo formulario `Post`, tenemos que llamar a `PostForm()` y pasar
 
 Tenemos que crear un archivo `post_edit.html` en el directorio `blog/templates/blog`. Para hacer que un formulario funcione necesitamos varias cosas:
 
-*   tenemos que mostrar el formulario. Podemos hacerlo, por ejemplo, con un simple `{{ form.as_p }}`.
+*   tenemos que mostrar el formulario. Podemos hacerlo, por ejemplo, con un simple `{% raw %}{{ form.as_p }}{% endraw %}`.
```

Comment:
lo probe con gitbook-editor.
en el tutorial en ingles, la version gitbook no se ve el tag raw porque ese es justo su proposito, pero si ves el codigo fuente de justo esas mismas lineas en el  tuto en ingles, si usan raw:

- we have to display the form. We can do that for example with a simple `{% raw %}{{ form.as_p }}{% endraw %}`.
[...]
- and finally just after the opening `<form ...>` tag we need to add `{% raw %}{% csrf_token %}{% endraw %}`. This is very important, since it makes your forms secure! Django will complain if you forget about this bit if you try to save the form:

El tuto en español ya usaba raw en el texto mencionando csrf_token (la linea de abajo) y si se ve en el tuto en español. Si ves las lineas arriba donde menciona el form.as_p ahi no se usaba el raw (que es donde yo hice mi cambio) y la falta del raw es justamente lo que provoca que no se vea el texto del form.as_p cuando lo miras en gitbook.